### ISSUE TITLE
The issue of accuracy in the reproduction model

### ISSUE THREAD

**Issue by MMM2024** [ayumiymk/DiG#3]
Thank you for sharing your work, it has inspired me a lot.
I recently used the pre-trained model you provided to fine-tune on STD and ARD respectively, using the same parameters as you did in the readme file. However, the accuracy of the evaluating results is very different from the accuracy given in your paper. What is the reason for this? Has the model structure of the open source code been updated compared to your paper?
Thanks again for your work and contribution. I look forward to your reply.

**Comment by ayumiymk** [comment]
Hi, the given pre-trained model is trained with 10 epochs while 3 epochs are in the paper. I also show the performance fine-tuned on the released pre-trained model in the ReadME. You can check it.
